packages/react-components/react-tree/src/components/TreeItem/useTreeItem.tsx

```diff
@@ -189,16 +214,32 @@ export function useTreeItem_unstable(props: TreeItemProps, ref: React.Ref<HTMLDi
       role: 'treeitem',
       'aria-level': level,
       [dataTreeItemValueAttrName]: value,
+      'aria-checked': selectionMode === 'multiselect' ? checked : undefined,
+      'aria-selected': selectionMode === 'single' ? checked : undefined,
```

Comment:
why does the aria attribute change depending on the selection mode?

Reply:
ARIA documentation indicates a preference for using `aria-checked` on multi selection and `aria-selected` on single selection